xml/x11_fonts.xml

```diff
@@ -244,12 +244,12 @@ DejaVuSansMono.ttf: "DejaVu Sans Mono" "Book"</screen>
      xlink:href="http://download.opensuse.org/repositories/M17N:/fonts/">M17N:fonts</link>
      repository. Add the repository to your list using the following command.
      For example, to add a repository for <phrase os="sles;sled">&slea;
-     15</phrase><phrase os="osuse">&productname; 15.2</phrase>:
+     15</phrase><phrase os="osuse">&productname; 15.&product-sp;</phrase>:
```

Comment:
I don't think these kinds of entity uses will make life/maintenance any easier. Can't we use `&productnumber;`? Or is that not yet profiled?